README.md

```diff
@@ -48,7 +48,9 @@ Output team
 
 ### Generating a gatorgrade.yml file
 
-Generate team
+The generation of yml file can be used to configure tests.
+
```

Comment:
Could you please try to simplify the messages I show below? They are what users need to know for calling the implementation function.
     Warning: User should provide precise names of folders or files and provide correct character case as input
     Warning: The empty folder or the folders only containing files starting with __ or . will be ignore
     Warning: Don't name folder or file starting with dot or double underscore. They will be ignored
     If you input a folder name, all the sub-dirs and files will be reserved except the ones starting with . or __
     Warning: Only folder names in root dir or the one after root will be checked. Don't put the target folders too deep
     The file names have no above problem. You can put it in any dir.

Reply:
Also, rn here is another input argument. User can provide relative directory to decide where get started to scan